hazelcast/src/main/java/com/hazelcast/internal/config/AbstractConfigLocator.java

```diff
@@ -165,6 +165,21 @@ protected boolean loadConfigurationFromClasspath(String configFileName) {
         }
     }
 
+    protected boolean loadConfigurationFromClasspath(String configFilePrefix, Collection<String> acceptedSuffixes) {
+        requireNonNull(acceptedSuffixes, "Parameter acceptedSuffixes must not be null");
+        checkFalse(acceptedSuffixes.isEmpty(), "Parameter acceptedSuffixes must not be empty");
+
+        for (String suffix : acceptedSuffixes) {
+            requireNonNull(suffix, "Parameter acceptedSuffixes must not contain nulls");
```

Comment:
Minor: since this is all internal API, maybe the not-null/not-empty checks are not really necessary here.

Reply:
I would like to leave them in, since they're also present in `loadFromSystemProperty(String propertyKey, boolean failOnUnacceptedSuffix, Collection<String> acceptedSuffixes)` which I didn't add or amend.